force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -258,12 +287,9 @@ public with sharing class VariablesDataTypesOperators {
      * @return The quotient as a casted (round down) Integer.
      */
     public static Integer performDivisionAndCast(Double a, Double b) {
-        // Perform the division and cast (round down) off the result.
-        Double divisionResult = null;
-
-        // Write the code for type casting the divisionResult to an Integer
-        Integer roundedResult = null;
-        return null; // Replace null with the variable you used to store the result
+        Double divisionResult = a / b;
+        Integer roundedResult = divisionResult.intValue();
```

Comment:
Great use of the intValue() method.
For practice, try casting it as an integer.